src/sxastarter/src/assets/sass/components/layout-container/_media.scss

```diff
@@ -0,0 +1,42 @@
+// Breakpoints variables
+$breakpoints-sm: 370px;
+$breakpoints-md: 768px;
+$breakpoints-lg: 1024px;
+$breakpoints-xl: 1280px;
+$breakpoints-xxl: 1536px;
+
+// Media queries
+@media only screen and (max-width: $breakpoints-sm) {
+  .slc-breakpoints-sm {
+    flex-direction: column;
+    align-items: stretch;
+  }
+}
+
+@media only screen and (min-width:calc(#{$breakpoints-sm + 1})) and (max-width: $breakpoints-md) {
```

Comment:
Limiting stack with min-width makes no sense. For example when you want to stack at for "Tablet you want it to be stacked for all the screens that less than Tablet  down to 0. You don't want to stack it only for Tablet  but than un-stack for a Phone